opencti-platform/opencti-graphql/src/migrations/1715067563363-fix-csv-ingester-without-mapper.js

```diff
@@ -0,0 +1,32 @@
+import { logApp } from '../config/conf';
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { listAllEntities } from '../database/middleware-loader';
+import { ENTITY_TYPE_INGESTION_CSV } from '../modules/ingestion/ingestion-types';
+import { elDeleteElements, elFindByIds } from '../database/engine';
+
+const message = '[MIGRATION] delete CSV feeds that have inexisting CSV mapper';
+
+export const up = async (next) => {
+  const context = executionContext('migration');
+  logApp.info(`${message} > started`);
+
+  const allIngesters = await listAllEntities(context, SYSTEM_USER, [ENTITY_TYPE_INGESTION_CSV]);
+  const mappersUsedIds = allIngesters.map((ingester) => ingester.csv_mapper_id);
```

Comment:
minor suggestion : maybe add a condition on allIngesters length before this line : 
`if (allIngesters.length > 0)` 
otherwise, no need to continue.